### PR TITLE
migrate to app directory

### PR DESCRIPTION
**Pull request by Southclaws**

### CODE REVIEW

File: web/next.config.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 /** @type {import('next').NextConfig} */
 
 const nextConfig = {
+  experimental: {
```
Comment (leerob) [inline web/next.config.js:4]: This can be removed now

File: web/package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     "framer-motion": "^7.2.1",
     "identicon.js": "^2.3.3",
     "lodash": "^4.17.21",
-    "next": "^13.0.6",
+    "next": "^13.3.0",
```
Comment (leerob) [inline web/package.json:28]: By getting on the latest version, you'll get _a lot_ of bug fixes

File: web/src/app/layout.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+"use client";
```
Comment (leerob) [inline web/src/app/layout.tsx:1]: Your root layout should remain a server component (it's okay that your providers, which are client components, are children here)

File: web/src/app/page.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+"use client";
+import IndexPage from "./index-page";
+
+export default async function Page() {
+  return <IndexPage />;
+}
+
+// NOTE: won't work until this is a server page, and it can't be a server page
+// until Chakra fixes support for this new app directory stuff...
+// export const metadata = {
```
Comment (leerob) [inline web/src/app/page.tsx:10]: You could make your server component `page` basically just have metadata, data fetching, and then forward everything to a client component child to do the styling with chakra.

Comment (Southclaws) [reply web/src/app/page.tsx]: Makes sense, thanks!

File: web/src/app/layout.tsx (hunk 3 above)
Comment (Southclaws) [reply web/src/app/layout.tsx]: This was what was confusing me a lot early on, I'm looking forward to Chakra making SSR and progressive enhancement easier in future versions but this works for now 🎉

**Pull request merged**